testcontainers/src/clients/cli.rs

```diff
@@ -464,6 +392,59 @@ impl Drop for Client {
     }
 }
 
+pub trait RunViaCli<I: Image> {
+    fn run(self) -> Container<I>;
+}
+
+impl<I: Image> RunViaCli<I> for RunnableImage<I> {
+    fn run(self) -> Container<I> {
```

Comment:
`run` function moved from `Http` to `RunnableImage` it can be directly called on `RunnableImage`

Reply:
Can we move this function up in the file such that the diff is minimal? That would help reviewing a lot :)

Reply:
It should be better now. 👍